superset-frontend/src/views/CRUD/hooks.ts

```diff
@@ -420,6 +420,10 @@ export function useImportResource(
       const formData = new FormData();
       formData.append('formData', bundle);
 
+      const reexportText = t(
```

Comment:
in other case, you're going for uppercase constant names, e.g. `REPORT_TEXT` in case we want to match that pattern.